.github/workflows/integration-tests-amd.yml

```diff
@@ -122,7 +122,8 @@ jobs:
           pytest --capture=tee-sys -rfs python/tutorials/06-fused-attention.py
           pytest --capture=tee-sys -rfs -n 8 third_party/amd/python/test/ \
                 --ignore=third_party/amd/python/test/test_scalarize_packed_fops.py \
-                --ignore=third_party/amd/python/test/test_address_sanitizer.py
+                --ignore=third_party/amd/python/test/test_address_sanitizer.py \
+                --ignore=third_party/amd/python/test/test_gluon_gfx1250.py
```

Comment:
This just completely skips the test? Can we check the compilation side with something like `pytest .. -k test_compile`? You can install hip-python if needed in the script.